orderbook/api.go

```diff
@@ -44,9 +44,9 @@ func (api *api) CreateOrder(ctx context.Context, params models.CreateOrderParams
 		Nonce:              params.SeriesNonce.Int64(),
 		Series:             0, // TODO: Series 0 always?
 	}
-	makerTraits := orderbook.BuildMakerTraits(buildMakerTraitsParams)
+	makerTraits := internal.BuildMakerTraits(buildMakerTraitsParams)
 
-	order, err := orderbook.CreateLimitOrderMessage(params, makerTraits)
+	order, err := internal.CreateLimitOrderMessage(params, makerTraits, int(api.chainId))
```

Comment:
Let's split API and order message into separate functions and allow user to Build order message on it's own and to send it to API when it needed